### ISSUE TITLE
Unicode Character Errors

### ISSUE THREAD

**Issue by NSC-nwest** [liquibase/liquibase-redshift#57]
Not sure if this is a redshift-specific or more general problem with liquibase. 

On windows. JRE v14. Using liquibase v4.3.5, liquibase-redshift-4.3.5.jar, and redshift-jdbc42-2.0.0.4.jar

I am reading in a SQL file that is encoded in valid UTF-8. 

```
create or replace function test.fn_test () RETURNS VARCHAR
IMMUTABLE
AS $$
  SELECT '([^A-Za-zÀ-ÖØ-öø-ÿÂ-ẑĀ-ẕÃ-ᵶ''. -]+)';
$$ LANGUAGE sql
;
```

When I run liquibase to create this object, it gives me this instead (both in the database and using the "updateSql" command to view the sql that it plans to run:

```
create or replace function test.fn_test () RETURNS VARCHAR
IMMUTABLE
AS $$
  SELECT '([^A-Za-zÃ€-Ã–Ã˜-Ã¶Ã¸-Ã¿Ã‚-áº‘Ä€-áº•Ãƒ-áµ¶''. -]+)';

$$ LANGUAGE sql;
```

So it appears there is something wrong with how liquibase or the extension is parsing unicode code points?

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-1707) by [Unito](https://www.unito.io)

**Comment by NSC-nwest** [comment]
I have also tried explicitly telling it the encoding:

```
  <changeSet author="test" id="fn_test">
      <sqlFile encoding="utf8" path="fn_test.sql" relativeToChangelogFile="true"/>
  </changeSet>
```

**Comment by filipelautert** [comment]
Hi - this should have been fixed in core for release 4.27.0 .
